### ISSUE TITLE
Add a visual clue for events with/without alarms

### ISSUE THREAD

**Issue by MarcosBL** [nextcloud/calendar#1197]
### Steps to reproduce
1. Open calendar
2. No visual clue on which events have/doesn't have alarms set

### Expected behaviour
I would like to have a visual clue on which events already have an alarm set, along the lines of this (AgenDAV):

<img src="https://i.imgur.com/UVjx3IT.png"/>

### Actual behaviour
No visual clue

### Sample

For example, in **app.js**, at the end of 

````
eventRender: function eventRender(event, element) {
````

We could add something like:

````
if(event.getSimpleEvent().alarm.length) {
    element.css({ 'background-image': 'url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAAaAAAAGgBo+Z6PAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAC6SURBVCiRrdKhTgNBEMbx35HUIUguIIrlDQ5HVQmkogjehcfgXbCowwCtQVNRVVJTdaL+MHNwIXtkSfiSSTbzzX9msrukVWCJRZyzVOAGbcQ8Bz6JKS22ES1ecfzbpBdscNHLT/CB56HJ19F9mvAuw7vqEgc9s0KDpwRYh3eeAkvsovNPteGVKfBP+hdwlFH/VdNd7wwP2ON+ALrDIW7x2IFvOMrcskE19JXOfL9njXVmU6dY4R3jVMEnDh8iN6jojq8AAAAASUVORK5CYII=")', 'background-repeat': 'no-repeat', 'background-position': 'right', 'padding-right': '16px' });
}
````

or even better, apply a proper class.

That would give us something like this in Nextcloud:

<img src="https://i.imgur.com/kWFOwgY.png"/>

**Comment by MarcosBL** [comment]
May I try a PR with this simple approach ?

**Comment by georgehrke** [comment]
#1755 was merged, so this will be included in the upcoming 2.0 version.
